source/coll_db.f90

```diff
@@ -49,28 +53,28 @@ subroutine cdb_allocDB
   use mod_alloc
   use numerical_constants
 
-  call alloc(cdb_cName,     mNameLen, cdb_nColl, " ",     "cdb_cName")
-  call alloc(cdb_cNameUC,   mNameLen, cdb_nColl, " ",     "cdb_cNameUC")
-  call alloc(cdb_cMaterial, 4,        cdb_nColl, " ",     "cdb_cMaterial")
-  call alloc(cdb_cFamily,             cdb_nColl, -1,      "cdb_cFamily")
-  call alloc(cdb_cNSig,               cdb_nColl, zero,    "cdb_cNSig")
-  call alloc(cdb_cNSigOrig,           cdb_nColl, zero,    "cdb_cNSigOrig")
-  call alloc(cdb_cLength,             cdb_nColl, zero,    "cdb_cLength")
-  call alloc(cdb_cOffset,             cdb_nColl, zero,    "cdb_cOffset")
-  call alloc(cdb_cRotation,           cdb_nColl, zero,    "cdb_cRotation")
-  call alloc(cdb_cBx,                 cdb_nColl, zero,    "cdb_cBx")
-  call alloc(cdb_cBy,                 cdb_nColl, zero,    "cdb_cBy")
-  call alloc(cdb_cFound,              cdb_nColl, .false., "cdb_cFound")
+  call alloc(cdb_cName,     mNameLen, cdb_nColl, " ",           "cdb_cName")
+  call alloc(cdb_cNameUC,   mNameLen, cdb_nColl, " ",           "cdb_cNameUC")
+  call alloc(cdb_cMaterial, 4,        cdb_nColl, " ",           "cdb_cMaterial")
+  call alloc(cdb_cFamily,             cdb_nColl, 0,             "cdb_cFamily")
+  call alloc(cdb_cNSig,               cdb_nColl, cdb_defColGap, "cdb_cNSig")
+  call alloc(cdb_cNSigOrig,           cdb_nColl, cdb_defColGap, "cdb_cNSigOrig")
```

Comment:
didn't we say that the default collgap was 0.0?

Reply:
I think so? Can't remember now. I will change it to zero anyway.